applications/DeepSpeed-Chat/training/step3_rlhf_finetuning/training_scripts/README.md

```diff
@@ -7,9 +7,7 @@ If you don't have step 1 and step 2 models. You may simply try
 ``` bash
 --actor_model_name_or_path facebook/opt-1.3b --critic_model_name_or_path facebook/opt-350m
 ```
-⚡⚡⚡ When you use above script, please make sure you comment out the following such that it won't load the model weight from previous paths.
-```bash
-applications/DeepSpeed-Chat/training/utils/model/model_utils.py#L60
-```
+⚡⚡⚡ When you use above script, please make sure you set `rlhf_training` is False
```

Comment:
Good catch @zhangfanTJU. Could you add more context for other users to understanding where they should set rlhf_training to be false

Reply:
Thank you very much for your reply and suggestions. I have revised the readme. Additionally, I did not provide a specific line number as the code is constantly being updated.